src/main/java/ui/ActionBar.java

```diff
@@ -88,6 +72,47 @@ private boolean hourMismatchCheck() {
 				&& !parentUi.showOKCancelDialog("Hours mismatch", "Warning: The worked hours do not match the target working hours. Do you want to continue?"));
 	}
 
+	/**
+	 * This is the method that is called when the tex button is clicked. First, it
+	 * checks for a mismatch in worked time, then it prompts the user to save the
+	 * file as LaTeX. <br/>
+	 * Exposed functionality because of hotkeys.
+	 */
+	public void compileTexButtonClicked() {
+		if (hourMismatchCheck()) {
```

Comment:
My intuition says that a xyCheck() method should return true if everything is in order.
I understand that in this case, the method essentially checks for an error, so it is the other way around.